### PR TITLE
Move to new golangci, fix issues

### PR DESCRIPTION
**Pull request by igor-hnizdo**
Also, remove ancient deprecated linters, plus linters, that are already in the default set.

### CODE REVIEW

File: core/release.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build !debug
```
Comment (igor-hnizdo) [inline core/release.go:1]: these are the new syntaxes for new go versions, `go fmt` puts them there

File: server/http.go [hunk 2]
```diff
@@ -35,7 +36,9 @@ func New(
 	longWriter.Log("starting")
 
 	https := &http.Server{
-		Addr: "127.0.0.1:21325",
+		Addr:              "127.0.0.1:21325",
+		ReadHeaderTimeout: 5 * time.Second,
```
Comment (igor-hnizdo) [inline server/http.go:40]: this is a protection against some DDoS thing... I don't think it applies to us, but I added it anyway

File: usb/hidapi.go [hunk 3]
```diff
@@ -44,6 +45,7 @@ func (b *HIDAPI) Enumerate() ([]core.USBInfo, error) {
 	b.mw.Log("low level done")
 
 	for _, dev := range devs { // enumerate all devices
+		dev := dev
```
Comment (igor-hnizdo) [inline usb/hidapi.go:48]: we take pointer of something we range through... gosec shows it as an issue even when we don't use it in a different goroutine, but still, probably should be like this, for the future etc

File: usb/lowlevel/hidapi/hid.go [hunk 4]
```diff
@@ -69,8 +70,9 @@ type HidDeviceInfo struct {
 // for enumeration, causing crashes if called concurrently.
 //
 // For more details, see:
-//   https://developer.apple.com/documentation/iokit/1438371-iohidmanagersetdevicematching
```
Comment (igor-hnizdo) [inline usb/lowlevel/hidapi/hid.go:72]: just gofmt thing

**Review by tsusanka — APPROVED** [review]
All changes LGTM. Merging.

**Pull request merged**